src/config.ts

```diff
@@ -0,0 +1,116 @@
+import path from 'path';
+import {cosmiconfigSync} from 'cosmiconfig';
+import {validate} from 'jsonschema';
+import {merge} from 'lodash';
```

Comment:
Oh, didn't see this before. I'd really like to not use lodash. I'm assuming {...foo} won't cut it here?  In that case, https://www.npmjs.com/package/deepmerge seems to be small and popular.

Reply:
I normally avoid lodash too, but I noticed it was already a sub-dep so it was “free.” Was thinking adding a dep for this would be extra weight.

So out of curiosity, I checked, and I _think_ the only core dep that requires this is `@babel/core`. Not sure if we’d drop that with es-module-lexer, but if we dropped Babel altogether it’d really lighten the install.

Also noticed that lodash is a req for `np` which `@pika/pack` uses. Of course that’s irrelevant here since that’s a `devDep`; just a fun bit of trivia.

Anyways, yeah +1 in favor of `deepmerge`. Yes, to your point, you could use the spread operator, but IMO it was too much spreading that it made the code harder to follow, and if someone edited this and didn’t spread it perfectly it’d break something. I’m in favor of a utility that reflects the intent.

Reply:
Yea I know what you mean, but it's an easy hole to fall down. Next thing you know the original sub-dep requirement is gone but you're too deep into using it yourself that you can't get rid of it.

+1 for es-module-lexer to drop our requirement for @babel/traverse / @babel/parser, and then I've always played around with making @babel/core a peerDependency (install it yourself, error if not found when `--optimize` is run). Those deps together account for a good chunk of Snowpack's dep graph